ODLA/include/ODLA/odla_common.h

```diff
@@ -135,6 +135,13 @@ typedef void odla_void;
 
 //! \brief Return status
 typedef enum {
+  ODLA_DL_ERROR,
+  ODLA_FILE_NOT_EXIST,
+  ODLA_INVALID_PARAM,
+  ODLA_MEM_ERROR,
+  ODLA_NULLPTR,
+  ODLA_UNSUPPORTED_DATATYPE,
+  ODLA_UNSUPPORTED_DEVICE_TYPE,
   ODLA_SUCCESS,
```

Comment:
Add the new enums after ODLA_SUCCESS. 
So success will always == 0.

Reply:
agree